pkg/setting/setting.go

```diff
@@ -936,6 +937,12 @@ func (cfg *Cfg) Load(args CommandLineArgs) error {
 	cfg.PluginAdminEnabled = pluginsSection.Key("plugin_admin_enabled").MustBool(true)
 	cfg.PluginAdminExternalManageEnabled = pluginsSection.Key("plugin_admin_external_manage_enabled").MustBool(false)
 
+	catalogHidePlugins := pluginsSection.Key("plugin_catalog_hide_plugins").MustString("")
+	for _, plug := range strings.Split(catalogHidePlugins, ",") {
```

Comment:
**Question:** maybe a stupid one, but just out of curiosity, do we have any "guidelines" in GoLang based on how to name these iterated variables?

(Only asking because sometimes I see a single letter, e.g. `p` for `plugin`, sometimes the full name of the variable)

Reply:
No question is stupid. No, we do not really have any "guidelines" around this. It's not mentioned [here](https://golang.org/doc/effective_go) either. But in general short and/or descriptive names I would say. Probably similar as with JavaScript/Typescript.